script/1_0_2current_en_script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -71,25 +71,25 @@ You seem like a very capable person."
 <Page20>
 "Yes, I try my best not to screw up in front of you. To be honest, I'm actually rather clumsy if anything."
 <Page21>
-...Certainly, it's possible that she may actually be a bit of a klutz?
+...It's certainly possible that she may actually be a bit of a klutz.
 It's the same with her volunteer work.
-People that are good at dealing with things wouldn't do something that doesn't benefit them in the slightest like that.
+Capable individuals wouldn't do something like that which doesn't benefit them in the slightest. // 要領がいい人間 capable might be a bit of a stretch but also connects to line 70
```

Comment:
> 要領がいい人間というのは、ああいう一文の得にもならない事はやらないものだ。
A capable individual wouldn't go for something like that; there's absolutely nothing for them to gain.